runtime/doc/builtin.txt

```diff
@@ -4348,6 +4348,19 @@ getregionpos({pos1}, {pos2} [, {opts}])            *getregionpos()*
 		the offset of the character's first cell not included in the
 		selection, otherwise all its cells are included.
 
+		Apart from the options supported by |getregion()|, {opts} also
+		supports the following:
+
+			eol		If |TRUE|, indicate positions beyond
+					the end of a line with "col" values
+					one more than the length of the line.
```

Comment:
Was this intentionally chosen instead of `v:maxcol` ?

Reply:
This is consistent with `getpos()` etc.